### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by igalloway**
Give more context to RDDRONE-FMU and the rationale for the Zephyr port.
Note that a complete physical platform is available for both Drone and Rover development

### CODE REVIEW

**Review by dleach02 — APPROVED** [review]

File: boards/arm/rddrone_fmuk66/doc/index.rst [hunk 1]
```diff
@@ -6,15 +6,25 @@ NXP RDDRONE-FMUK66
...
+The RDDRONE FMUK66 is primarily designed as an autonomous drone "Flight Management Unit" control board which incorporates DroneCode standard commonly used peripheral
+connectors. The MCU onboard is a Kinetis K66. The FMUK66 hardware includes onboard IMU (Intertial measurement units), Ports for off the shelf GPS modules with integrated arming swtiches, LEDs and enunciators, SDCARD for logging, CAN ports, PWM ports, 100BaseT1 "2-wire" Ethernet, and SE050 Secure Element. 
+
+Note that historically this board is supported by another RTOS, NuttX, and the PX4 flight stack running on top, and can fly Multicopters, VTOL and traditional Aircraft, ground based rovers and water going vessels. 
+The intention here is to have a functional port to Zephyr RTOS so that experimentation and development of flight/Vehicle stacks can also begin with Zephyr.
+The FMUK66 hardware is suitable for use in other mobile robotics applications, or simply as hardware where where CAN, T1 Ethernet, or Secure development is needed.
+
+- The RDDRONE-FMUK66 kit comes with a J-Link Edu Mini for programming and UART console as well as a power distribution and measuring module as well as a GPS.
```
Comment (gmarull) [inline boards/arm/rddrone_fmuk66/doc/index.rst:16]: please, wrap to 80cols (as file originally did)

File: boards/arm/rddrone_fmuk66/doc/index.rst [hunk 2]
```diff
@@ -6,15 +6,25 @@ NXP RDDRONE-FMUK66
...
-- Comes with a J-Link Edu Mini for programming and UART console.
 
 .. image:: ./rddrone_fmuk66.jpg
    :width: 720px
    :align: center
    :alt: RDDRONE-FMUK66
+   
+Development kits incorporating RDDRONE-FMUK66
+********
+- NXP KIT-HGDRONEK66 is a complete quadcopter drone kit that incorporates the RDDRONE-FMUK66
```
Comment (gmarull) [inline boards/arm/rddrone_fmuk66/doc/index.rst:26]:
```suggestion
Development kits incorporating RDDRONE-FMUK66
*********************************************

- NXP KIT-HGDRONEK66 is a complete quadcopter drone kit that incorporates the RDDRONE-FMUK66
```

**Review by gmarull — CHANGES_REQUESTED** [review]
Please adjust commit message as per https://docs.zephyrproject.org/latest/contribute/#commit-guidelines

File: boards/arm/rddrone_fmuk66/doc/index.rst (hunk 1 above)
Comment (igalloway) [reply boards/arm/rddrone_fmuk66/doc/index.rst]: OK this is done

File: boards/arm/rddrone_fmuk66/doc/index.rst (hunk 2 above)
Comment (igalloway) [reply boards/arm/rddrone_fmuk66/doc/index.rst]: Sorry - i *think* i added this suggestion by clicking the "add suggestion to batch" button. But i'm not 100% certain.
I agree with the suggestion in any case.

**Review by mmahadevan108 — DISMISSED** [review]

**Comment by mbolivar-nordic** [comment]
@igalloway please rebase your changes instead of creating a separate commit fixing the line lengths.

@mmahadevan108 since you have +1 privileges, please take note of the above. The +1 should not have been given; we don't do "fixup" commits like this in Zephyr.

**Review by dleach02 — COMMENTED** [review]
@igalloway please address the requested changes. 

Also, update the commit title to be:

boards: arm: rddrone_fmuk66: Update documentation

**Comment by igalloway** [comment]
@dleach02  I combined the two commits and renamed them.

**Review by dleach02 — APPROVED** [review]

File: boards/arm/rddrone_fmuk66/doc/index.rst [hunk 3]
```diff
@@ -6,15 +6,37 @@ NXP RDDRONE-FMUK66
...
-- Comes with a J-Link Edu Mini for programming and UART console.
 
 .. image:: ./rddrone_fmuk66.jpg
    :width: 720px
    :align: center
    :alt: RDDRONE-FMUK66
+   
+Development kits incorporating RDDRONE-FMUK66
+********
+- NXP KIT-HGDRONEK66 is a complete quadcopter drone incorporating RDDRONE-FMUK66
```
Comment (gmarull) [inline boards/arm/rddrone_fmuk66/doc/index.rst:38]:
```suggestion
Development kits incorporating RDDRONE-FMUK66
*********************************************

- NXP KIT-HGDRONEK66 is a complete quadcopter drone incorporating RDDRONE-FMUK66
```

**Review by gmarull — CHANGES_REQUESTED** [review]

File: boards/arm/rddrone_fmuk66/doc/index.rst [hunk 4]
```diff
@@ -6,15 +6,37 @@ NXP RDDRONE-FMUK66
...
+and the PX4 flight stack running on top, and can fly Multicopters, VTOL and 
+traditional aircraft, ground based rovers and water going vessels. 
+The intention here is to have a functional port to Zephyr RTOS so that
+experimentation and development of flight/Vehicle stacks can also begin 
+with Zephyr.
+The FMUK66 hardware is suitable for use in other mobile robotics applications,
+or as hardware where where CAN, T1 Ethernet, or Secure development is needed.
+
+- The RDDRONE-FMUK66 kit comes with a J-Link Edu Mini for programming and UART
+ console as well as a power distribution and measuring module as well as a GPS.
```
Comment (dleach02) [inline boards/arm/rddrone_fmuk66/doc/index.rst:29]: did you intend for this to be a bullet item of one? Sphinx build is complaining about this:

":Bullet list ends without a blank line; unexpected unindent."

File: boards/arm/rddrone_fmuk66/doc/index.rst [hunk 5]
```diff
@@ -6,15 +6,37 @@ NXP RDDRONE-FMUK66
...
```
Comment (dleach02) [inline boards/arm/rddrone_fmuk66/doc/index.rst:28]:
```suggestion
 console as well as a power distribution and measuring module as well as a GPS.
 
```

**Pull request closed without merging**